WalletWasabi.Packager/Program.cs

```diff
@@ -75,23 +73,25 @@ private static async Task Main(string[] args)
 
 			if (DoPublish || OnlyBinaries)
 			{
-				Publish();
+				await PublishAsync().ConfigureAwait(false);
 
 				IoHelpers.OpenFolderInFileExplorer(BinDistDirectory);
 			}
 
+#pragma warning disable CS0162 // Unreachable code detected
 			if (!OnlyBinaries)
 			{
 				if (DoSign)
 				{
-					Sign();
+					await SignAsync().ConfigureAwait(false);
 				}
 
 				if (DoRestoreProgramCs)
 				{
 					RestoreProgramCs();
 				}
 			}
+#pragma warning restore CS0162 // Unreachable code detected
```

Comment:
If I delete this line, no warning comes up.

Reply:
This is "restore" part. You need to delete both lines and then the warnings are visible.

Reply:
If I delete both pragma lines, then of course, the warnings are visible. 

Did you mean to "wrap" that code block between the pragma lines, so it's clearer?
If yes, then i'm fine with it.

Reply:
Yes, I just wrap only the necessary part and not much larger part unnecessarily.